packages/containerd-1.7/containerd.service

```diff
@@ -3,6 +3,7 @@ Description=containerd container runtime
 Documentation=https://containerd.io
 After=network-online.target configured.target
 Wants=network-online.target configured.target
+Requires=snapshotters-configured.service
```

Comment:
If you want containerd to start after `snapshotters-configured.servce`, you should add the service to the `After=` of this unit.

Also the name of your service is wrong: `configure-snapshotter.service` is what you used in the first commit.